bundles/org.openhab.binding.velbus/src/main/java/org/openhab/binding/velbus/internal/handler/VelbusBridgeHandler.java

```diff
@@ -141,7 +153,7 @@ public void unregisterRelayStatusListener(byte address, VelbusPacketListener pac
     @Override
     public void dispose() {
         if (serialPort != null) {
-            serialPort.disconnect();
+            serialPort.close();
```

Comment:
Should probably remove the listener too just to make sure cleanup is complete.
```suggestion
            serialPort.removeEventListener(this);
            serialPort.close();
```